File: fieldsync-inventory-backend/src/main/java/com/example/fieldsync_inventory_api/dto/sales/customer/CustomerRequestDTO.java
package com.example.fieldsync_inventory_api.dto.sales.customer;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
@Builder
public class CustomerRequestDTO {
    private Integer customerTypeId;
    private String fullName;
    private String email;
    private String phone;
    private String address;
}

File: fieldsync-inventory-app/app/src/main/java/com/example/fieldsync_inventory_app/data/remote/dto/customer/CustomerRequestDto.kt
package com.example.fieldsync_inventory_app.data.remote.dto.customer

// Request DTO (remains the same as it only needs the ID for creation/update)
data class CustomerRequestDto(
    val customerTypeId: Int,
    val fullName: String,
    val email: String?,
    val phone: String?,
    val address: String?
)